APMrover2/Rover.h

```diff
@@ -490,6 +491,15 @@ class Rover : public AP_HAL::HAL::Callbacks {
 
     // Simple mode
     float simple_sin_yaw;
+
+    enum FS_Options {
+        OPTION_HOLD_FS_ALLOW =  (1<<0)
+    };
+
+    enum Rtl_End_Behave {
```

Comment:
let's move RTL_End_Behave up under cruise_learn_t and rename it to "RTL_Done_Behave" (note capital T and L) to be more consistent with the existing Mis_Done_Behave enum.  Perhaps just rename the options to "HOLD" and "LOITER" or maybe "RTL_DONE_BEHAVE_HOLD", etc.  sorry to be picky.

Reply:
done....I think